pkg/controller/predicates/resources/resources.go

```diff
@@ -117,3 +118,27 @@ var DSCComponentUpdatePredicate = predicate.Funcs{
 		return false
 	},
 }
+
+var DSCIReadiness = predicate.Funcs{
+	UpdateFunc: func(e event.UpdateEvent) bool {
+		oldObj, ok := e.ObjectOld.(*dsciv1.DSCInitialization)
+		if !ok {
+			return false
+		}
+		newObj, ok := e.ObjectNew.(*dsciv1.DSCInitialization)
+		if !ok {
+			return false
+		}
+
+		return oldObj.Status.Phase != newObj.Status.Phase
+	},
+	CreateFunc: func(e event.CreateEvent) bool {
+		return false
+	},
+	DeleteFunc: func(e event.DeleteEvent) bool {
+		return true
+	},
```

Comment:
As talked over chat, this is probably not needed.

Reply:
done